src/app.js

```diff
@@ -31,10 +31,10 @@ export function app() {
 	const handlersByTopic = new Map();
```

Comment:
`TODO` for later:

if we have `EVENT`s, then maybe `topic` is not the right word, it should probably be `name` or `type`.


as a sidenote: `data` could stay as is, or be renamed to `payload`.